utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -580,6 +580,11 @@ def create_argument_parser():
                 'module-only targets on Darwin platforms. These targets are '
                 'in addition to the full library targets.')
 
+    option(['--skip-build-stdlib-when-cross-compiling'],
+           toggle_false('build_stdlib_when_cross_compiling'),
+           help='Don\'t build the standard library when building '
+                'the Swift compiler for the cross compile hosts')
```

Comment:
Using double quotes would avoid the need to escape the quote.
```suggestion
           help="Don't build the standard library when building "
                "the Swift compiler for the cross compile hosts")
```